src/utils/detect-server.js

```diff
@@ -55,14 +55,17 @@ module.exports.serverSettings = async (devConfig, flags, log) => {
   } else if (devConfig.framework === '#static') {
     settings.framework = devConfig.framework
   } else {
-    const detectorName = detectorsFiles.find(dt => `${dt}.js` === devConfig.framework)
-    if (!detectorName) throw new Error('Unsupported value provided for "framework" option in config')
-
+    // detectorsFiles end in '.js`; but devConfig.framework shouldn't. To test for match, append '.js' to devConfig.framework.
+    const detectorName = detectorsFiles.find(dt => dt === `${devConfig.framework}.js`)
+    if (!detectorName) throw new Error(`Unsupported value '${devConfig.framework}' provided for "framework" option in config`)
+    
     const detector = loadDetector(detectorName)
     const detectorResult = detector()
     if (!detectorResult) throw new Error(`Specified "framework" detector "${devConfig.framework}" did not pass requirements for your project`)
 
     settings.args = chooseDefaultArgs(detectorResult.possibleArgsArrs)
+    // Merge detectorResult default settings into settings Object.
+    Object.assign(settings, detectorResult)
```

Comment:
Not sure if this is right; but it didn't appear we were really using the `detectorResult` fully (`chooseDefaultArgs()` looks like it doesn't pull in all the defaults that we want).